packages/stores/types.ts

```diff
@@ -197,6 +197,7 @@ type TCommonStore = {
     platform: string;
     routeBackInApp: (history: Pick<RouteComponentProps, 'history'>, additional_platform_path?: string[]) => void;
     routeTo: (pathname: string) => void;
+    server_time?: moment.Moment;
```

Comment:
```suggestion
    server_time: moment.Moment;
```
@shafin-deriv It's not optional, It has a default value in the `CommonStore` 🤔

Reply:
@shafin-deriv Also please update the `mockStore` with a default value for it 🙏🏻